src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,55 @@
+var timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
+var currencies = ['usd']
+
+function optionsChecker (options) {
+  if ('currency' in options) {
+    if (typeof options.currency === 'undefined') {
+      options.currency = 'usd'
+    }
+    if (currencies.indexOf(options.currency) === -1) {
+      throw new Error('This is not a valid currency')
+    }
+  }
+  if ('timeBound' in options) {
+    if (typeof options.timeBound === 'undefined' || options.timeBound === '') {
+      options.timeBound = ''
+    }
+    if (timeBound.indexOf(options.timeBound) === -1) {
+      throw new Error('This is not a valid timebound suffix')
+    }
+  }
+}
+
+function metricNameGenerator (metric, options) {
+  var metricName = metric
+
+  if (currencies.indexOf(options.currency) >= 0 && options.currency !== '') {
+    metricName += ('_' + options.currency)
+  }
+
+  if (timeBound.indexOf(options.timeBound ) >= 0 && options.timeBound !== '') {
```

Comment:
reversing the check makes more logical sense: `if it's not empty and is supported`. Ideally, if you drop the `''` from the supported list you can just test for `indexOf`

Reply:
Is there a difference if I reverse them in the parentheses? And I really prefer if I left it the way it is right now, with '' in the timeBound, because '' really is '20y', it's something like a timeBoundSuffix